lib/apiservers/engine/backends/container.go

```diff
@@ -915,13 +917,18 @@ func (c *Container) ContainerInspect(name string, size bool, version version.Ver
 		}
 	}
 
-	inspectJSON, err := containerInfoToDockerContainerInspect(name, results.Payload)
+	inspectJSON, err := containerInfoToDockerContainerInspect(vc, results.Payload)
```

Comment:
I know `vc` is checked in `containerInfoToDockerContainerInspect`, but I think it would be better to check here as well, in case someone in the future adds code further down that uses `vc`.